modules/prefixer.js

```diff
@@ -99,11 +99,22 @@ if (ExecutionEnvironment.canUseDOM) {
   domStyle = document.createElement('p').style;
 
   // Based on http://davidwalsh.name/vendor-prefix
+  var cssVendorPrefix;
+  var prefixMatch;
   var windowStyles = window.getComputedStyle(document.documentElement, '');
-  var prefixMatch = Array.prototype.slice.call(windowStyles)
-    .join('')
-    .match(/-(moz|webkit|ms|o)-/);
-  var cssVendorPrefix = prefixMatch && prefixMatch[0];
+
+  // Array.prototype.slice.call(windowStyles) fails with
+  // "Uncaught TypeError: undefined is not a function"
+  // in older versions Android (KitKat) web views
+  for (var i = 0; i < windowStyles.length; i++) {
+    prefixMatch = windowStyles[i].match(/-(moz|webkit|ms|o)-/);
```

Comment:
No problem. Where do you prefer constants to be defined? Top of the file, top of method, constant's file, etc?